src/main/scala/chisel3/iotesters/ChiselMain.scala

```diff
@@ -156,7 +156,7 @@ object chiselMain {
     context.backend = Some(context.backendType match {
       case "firrtl" =>
         val file = new java.io.File(context.targetDir, s"${dut.name}.ir")
-        val ir = io.Source.fromFile(file).getLines mkString "\n"
+        val ir = scala.io.Source.fromFile(file).getLines mkString "\n"
```

Comment:
Since you are changing this anyway, seems like a good time to 
```suggestion
        val ir = FileUtils.getText(file)
```

Reply:
Makes sense to me. Fixed in the rebase.